### PR TITLE
Encryption common

### PR DESCRIPTION
**Pull request by alexemc**
# Description
Added references to Encryption from the common pages.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/_index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ menu:
 no_list: true
 ---
 
-The Dell Technologies (Dell) Container Storage Modules (CSM) enables simple and consistent integration and automation experiences, extending enterprise storage capabilities to Kubernetes for cloud-native stateful applications. It reduces management complexity so developers can independently consume enterprise storage with ease and automate daily operations such as provisioning, snapshotting, replication, observability, authorization and, resiliency.
+The Dell Technologies (Dell) Container Storage Modules (CSM) enables simple and consistent integration and automation experiences, extending enterprise storage capabilities to Kubernetes for cloud-native stateful applications. It reduces management complexity so developers can independently consume enterprise storage with ease and automate daily operations such as provisioning, snapshotting, replication, observability, authorization, application mobility, encryption and, resiliency.
```
Comment (rsedlock1958) [inline content/docs/_index.md:12]: Change the end of the sentence to read "...encryption, and resiliency."

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. There is one minor edit.

**Review by anathoodell — DISMISSED** [review]

File: content/docs/_index.md (hunk 1 above)
Comment (alexemc) [reply content/docs/_index.md]: Addressed. Please review and approve the change.

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No further edits are required.

**Review by anathoodell — APPROVED** [review]

**Review by bpjain2004 — APPROVED** [review]

**Pull request merged**